akka-docs-new/src/main/paradox/guide/actors-intro.md

```diff
@@ -0,0 +1,259 @@
+# What problems does the actor model solve? 
+
+Akka uses the actor model to overcome the limitations of traditional object-oriented programming models and meet the 
+unique challenges of highly distributed systems. To fully understand why the actor model is necessary, it helps to 
+identify mismatches between traditional approaches to programming and the realities of concurrent and distributed 
+computing. 
```

Comment:
I would add one sentence about the usual "free lunch is over" that we need to use many threads and systems to build the kind of systems that people expect nowadays